<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buchungsdetails</class>
 <widget class="QMainWindow" name="Buchungsdetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ID_Buchungsdetails">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Preis</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_Preis_Buchungsdetails">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Von</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_Von_Buchungsdetails">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bis</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit_Bis_Buchungsdetails">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_Buchungstyp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>371</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset>
       <normaloff>../build-newPraktikum3-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/train.jpg</normaloff>../build-newPraktikum3-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/train.jpg</iconset>
     </attribute>
     <attribute name="title">
      <string>Zug</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>von</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Von_Zug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_Abfahrt_Zug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Abfahrt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>nach</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Nach_Zug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ankunft</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_Ankunft_Zug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TicketTyp</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Tickettyp_Zug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>180</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>über</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_Uber_Zug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>256</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset>
       <normaloff>../build-newPraktikum3-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/car.jpg</normaloff>../build-newPraktikum3-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/car.jpg</iconset>
     </attribute>
     <attribute name="title">
      <string>Mietwagen</string>
     </attribute>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>VehicleClass</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Vehicleclass_Car_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Abholort</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Rückgabeort</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Unternehmen</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Abholort_Car__Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Ruckgabeort_Car_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>170</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Unternehmen_Car_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>240</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="icon">
      <iconset>
       <normaloff>../build-newPraktikum3-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/hotel.jpg</normaloff>../build-newPraktikum3-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/hotel.jpg</iconset>
     </attribute>
     <attribute name="title">
      <string>Hotel</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RoomTyp</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Stadt</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Hotel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Roomtyp_Hotel_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Stadt_Hotel_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>160</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Hotel_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="icon">
      <iconset>
       <normaloff>../build-newPraktikum3-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/flight.png</normaloff>../build-newPraktikum3-Desktop_Qt_6_5_0_MinGW_64_bit-Debug/flight.png</iconset>
     </attribute>
     <attribute name="title">
      <string>Flug</string>
     </attribute>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BookingClass</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Abfahrt in</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>290</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ankunft in</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Airline</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Bookingclass_Flug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>160</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Abfahrt_Flug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Ankunft_Flug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>290</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Airline_Flug_Buchungsdetails">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>360</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>iatacodes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_iatacodes">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Flughafen_name">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>291</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_OK">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>620</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>393</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
